dgraphtest/cluster.go

```diff
@@ -68,15 +68,15 @@ type GraphQLParams struct {
 	Variables map[string]interface{} `json:"variables"`
 }
 
-type graphQLResponse struct {
+type GraphQLResponse struct {
 	Data       json.RawMessage        `json:"data,omitempty"`
 	Errors     x.GqlErrorList         `json:"errors,omitempty"`
 	Extensions map[string]interface{} `json:"extensions,omitempty"`
 }
 
-func (hc *HTTPClient) LoginIntoNamespace(user, password string, ns uint64) error {
+func (hc *HTTPClient) LoginIntoNamespace(user, password string, ns uint64, refreshToken string) error {
```

Comment:
the refreshtoken is already available inside the HTTPClient object, do we still need to pass it?

Reply:
We need a function which can log a user into the namespace using the refreshjJWT token

Reply:
could this argument be a boolean then that whether to use the refreshtoken or username password? We could also create another function that Logs in using the refresh token.

Reply:
Created a new function